test/unit/main/post/views/filter-by-datasource.spec.js

```diff
@@ -90,5 +90,12 @@ describe('filter by datasource directive', function () {
             isolateScope.showOnlyIncoming('SMS');
             expect($location.path()).toEqual('/views/data');
         });
+        it('should toggle the filters based on if selected filter is activated or not', function () {
+            console.info(isolateScope.filters.source);
```

Comment:
Should you remove the console.info here?

Reply:
Hehe, yes I should, fixed now...